jmx-metrics/docs/target-systems/kafka.md

```diff
@@ -7,116 +7,105 @@ These metrics are sourced from Kafka's exposed Yammer metrics for each instance:
 
 ### Broker Metrics
 
-* Name: `kafka.messages.in`
-* Description: Number of messages in per second
-* Unit: `1`
-* Instrument Type: LongValueObserver
+* Name: `kafka.message.count`
+* Description: The number of messages received by the broker
+* Unit: `messages`
```

Comment:
As was brought to my attention in https://github.com/open-telemetry/opentelemetry-java-contrib/pull/210#discussion_r789912075 the units should be in `{annotation}` form: https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units

Reply:
Will resolve.